wbooks/src/app/style.ts

```diff
@@ -0,0 +1,9 @@
+import { Dimensions, StyleSheet } from 'react-native';
+
+const { width } = Dimensions.get('screen');
+
+const styles = StyleSheet.create({
+  header: { padding: 56, width }
```

Comment:
No, this is not ok

3 things:
* This style must be in the `HeaderBackground` screen
* The style is `  image: { resizeMode: 'stretch', width: '100%' }`
* The other header styles must be inside `screenOptions`